src/GAP.jl

```diff
@@ -253,13 +255,22 @@ function __init__()
 
     real_JuliaInterface_path[] = Setup.locate_JuliaInterface_so(sysinfo)
 
-    gaproots = sysinfo["GAPROOTS"]
-    cmdline_options = ["", "-l", gaproots]
+    roots = [
+            # GAP root with custom sysinfo.gap
+            Setup.gaproot(),
+            # GAP root for the the actual GAP library, from GAP_lib_jll
+            abspath(GAP_lib_jll.find_artifact_dir(), "share", "gap"),
+            # GAP root into which PackageManager installs packages by default
+            Packages.gap_pkgdir(),
+            ]
+    cmdline_options = ["", "-l", join(roots, ";")]
 
     # tell GAP about all artifacts that contain GAP packages
     pkg_artifacts = filter(startswith("GAP_pkg_"), keys(TOML.parsefile(find_artifacts_toml(@__FILE__))))
-    pkgdirs = join((realpath(@artifact_str(name)) for name in pkg_artifacts), ';')
-    append!(cmdline_options, ["--packagedirs", pkgdirs])
+    pkgdirs = [realpath(@artifact_str(name)) for name in pkg_artifacts]
+    push!(pkgdirs, abspath(@__DIR__, "..", "pkg", "JuliaInterface"))
+    push!(pkgdirs, abspath(@__DIR__, "..", "pkg", "JuliaExperimental"))
```

Comment:
```suggestion
    push!(pkgdirs, abspath(@__DIR__, "..", "pkg"))
```
alternatively, you could do this, but there is no difference whatsoever as no more packages are put in `../pkg/`